src/google-maps/map-event-manager.ts

```diff
@@ -18,11 +19,11 @@ export class MapEventManager {
   /** Pending listeners that were added before the target was set. */
   private _pending: {observable: Observable<any>, observer: Subscriber<any>}[] = [];
   private _listeners: google.maps.MapsEventListener[] = [];
-  private _target: MapEventManagerTarget;
+  private _targets = new BehaviorSubject<MapEventManagerTarget>(undefined);
```

Comment:
I think this member name is a little confusing. It implies there are several targets at the same time, when in fact there is still just a single target.

Reply:
Also, should the type be BehaviorSubject<MapEventManagerTarget|undefined> ?

Reply:
My thinking with making it plural was that it's a "stream of targets", but since we don't follow the `$` suffix convention for observables, there wasn't a better way to indicate it. I'll rename it.

As for the `| undefined`, it's not necessary since `MapEventManagerTarget` includes `undefined` already.